### ISSUE TITLE
Feature Request: Move Agent Across Group

### ISSUE THREAD

**Issue by openmoto** [Ylianst/MeshCentral#85]
Would it be possible to add feature allowing user to move agent from one group to another.
This would allow bulk install with one agent then sort machines across groups.
Also helpful for creating a temporary group to give someone access to a machine or group of machines without giving access the entire group(s).

**Comment by Ylianst** [comment]
Hi. When clicking on a device, you can set "Tags". When, go back to the devices view and group by tags. There is no way right now to grant access based on tags, but it's an easy way to group devices.

**Comment by jcam7044** [comment]
I was going to make a request for this very same feature. That was the first thing I tried to do when I setup my first couple agents.  DWService has a project that is similar to this one that has this feature.  I think they may actually be using the same agent. https://github.com/dwservice/agent

**Comment by Ylianst** [comment]
After thinking about it a bit... I will do this. I can see how I can do it. I got a pile of meetings on Monday but hopefully I can get it done this week.

**Comment by Ylianst** [comment]
Just published MeshCentral 0.2.7-y with group change feature. Click on a device and on the bottom right you will see "Change Group". Please test it plenty and let me know if you see any issues. Thanks.

**Comment by openmoto** [comment]
Thanks!
Works great. Any chance of adding ability to do this to multiple computers from the 
My Devices screen? 
The current solution works for me but for people managing a lot of computers, moving selected computers to group would be more convenient.

Thanks

**Comment by sorvani** [comment]
I was just coming here to post about this. Awesome.

It works great.

**Comment by Ylianst** [comment]
Yes, I need to make moving computers a "Group Action" on the main screen. It's on the TODO list now.

**Comment by Ylianst** [comment]
Just published [MeshCentral v0.2.8-v](https://www.npmjs.com/package/meshcentral) with batch move of devices between groups. Check a few devices, click the "Group Action" button, select "Move group" (it will be "Move to device group" in the next version). Hit ok, select a group and hit ok again.

I also did a odd thing in this version. If you delete a device group and a device connect to a deleted group, the group will undelete itself. I did this because people would delete a group and devices that would connect would have nowhere to go. This fixes it well I think.
